synapse/storage/databases/main/events.py

```diff
@@ -871,6 +871,29 @@ def event_dict(event):
                     updatevalues={"have_censored": False},
                 )
 
+        state_events_and_contexts = [
+            ec for ec in events_and_contexts if ec[0].is_state()
+        ]
```

Comment:
Should we update the docstring of this method? It already seems outdated TBH.

Reply:
@erikjohnston Did you see this comment?

Reply:
woops